indico/modules/events/abstracts/settings.py

```diff
@@ -129,6 +129,7 @@ class BOALinkFormat(RichEnum):
     'scale_upper': 5,
     'allow_comments': True,
     'allow_convener_judgment': False,  # whether track conveners can make a judgment (e.g. accept/reject)
+    'allow_convener_track_change': True,
```

Comment:
should default to False IMO

Reply:
If I default it to false, I'm afraid current judges won't be able to edit the tracklist as before. Since now the two options (judging or changing tracks) are independent.

Reply:
but judges/managers should always be able to do that regardless of this setting

Reply:
so the current logic change is wrong - maybe it should be a new `abstract.can_track_change()` method which then checks for can_judge or (if the setting is enabled) can_convene?

Reply:
That is another option, even though we would have to change the convener_track_change option to be dependent on whether or not the previous (judgment) is selected.

Reply:
I'd just hide it when convener judgment is enabled; `HiddenUnless` should do the job